docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -1,184 +1,136 @@
 
 ```shell
-$ while true; do curl http://146.148.6.244; done
+$ kubectl run -i --tty load-generator --image=busybox /bin/sh
+
+Hit enter for command prompt
+
+$ while true; do wget -q -O- http://php-apache.default.svc.cluster.local; done
 ```
 
-We may examine, how CPU load was increased (the results should be visible after about 3-4 minutes) by executing:
+We may examine, how CPU load was increased by executing (it usually takes 1 minute):
 
 ```shell
 $ kubectl get hpa
-NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   50%       305%      1         10        4m
+NAME         REFERENCE                     TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   Deployment/php-apache/scale   50%       305%      1         10        3m
+
 ```
 
 In the case presented here, it bumped CPU consumption to 305% of the request.
-As a result, the replication controller was resized to 7 replicas:
+As a result, the deployment was resized to 7 replicas:
 
 ```shell
-$ kubectl get rc
-CONTROLLER   CONTAINER(S)   IMAGE(S)                               SELECTOR         REPLICAS   AGE
-php-apache   php-apache     gcr.io/google_containers/hpa-example   run=php-apache   7          18m
+$ kubectl get deployment php-apache
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+php-apache   7         7         7            7           19m
 ```
 
-Now, we may increase the load even more by running yet another infinite loop of queries (in yet another terminal):
+## Step Four: Stop load
 
-```shell
-$ while true; do curl http://146.148.6.244; done
-```
+We will finish our example by stopping the user load.
 
-In the case presented here, it increased the number of serving pods to 10:
+We will terminate infinite ``while`` loop sending requests to the server and verify the result state:
 
 ```shell
 $ kubectl get hpa
-NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   50%       65%       1         10        14m
+NAME         REFERENCE                     TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   Deployment/php-apache/scale   50%       0%        1         10        11m
 
-$ kubectl get rc
-CONTROLLER   CONTAINER(S)   IMAGE(S)                               SELECTOR         REPLICAS   AGE
-php-apache   php-apache     gcr.io/google_containers/hpa-example   run=php-apache   10         24m
+$ kubectl get deployment php-apache
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+php-apache   1         1         1            1           27m
 ```
 
-## Step Four: Stop load
+As we see, in the presented case CPU utilization dropped to 0, and the number of replicas dropped to 1.
 
-We will finish our example by stopping the user load.
+## Appendix: Other possible scenarios
```

Comment:
How about adding something like FAQ?

* I have high utilization but HPA doesn't increase replica count. What happened? - something about min/max, quota, pending pods etc.
* It takes forever to adjust number of replicas. Why? - something about stabilization and heapster latency
* How should I choose target utilization? - something about load tests or default 80%
* How can I see why HPA controller decided to change replica count? - something about debugging